apps/astarte_appengine_api/test/support_v2/cases/device.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Astarte.Test.Cases.Device do
+  use ExUnit.CaseTemplate
+  use Astarte.Test.Cases.Interface
+
+  using do
+    quote do
+      # Do nothing for now
+    end
+  end
+
+  alias Astarte.Test.Setups.Device, as: DeviceSetup
+
+  setup_all [
+    {DeviceSetup, :init},
+    {DeviceSetup, :setup}
+  ]
```

Comment:
Note that `setup_all` runs once before all tests in a case. In order to improve test isolation and clarity, it would be better to generate device data for each test (i.e. with `setup`).
The same goes also for groups and interfaces